clients/python/client/osparc/_studies_api.py

```diff
@@ -36,3 +58,53 @@ def create_study_job(self, study_id, job_inputs, **kwargs):
     def clone_study(self, study_id, **kwargs):
         kwargs = {**kwargs, **ParentProjectInfo().model_dump(exclude_none=True)}
         return super().clone_study(study_id, **kwargs)
+
+    def studies(self) -> PaginationGenerator:
+        def pagination_method():
+            page_study = super(StudiesApi, self).list_studies(limit=20, offset=0)
+            assert isinstance(page_study, PageStudy)  # nosec
```

Comment:
Hmm, yeah I am also not happy with this. But it originates from the fact that our automatically generated client isn't doing a good job at the type annotations. I think a more useful thing to do would be to upgrade the autogeneration tool to the latest version. That one uses pydantic, so I suspect the type annotation should be significantly better. I once tried to generate the client with the latest version of the tool and it worked more or less out of the box. So it might be a low hanging fruit.